docs/reference/frozen-indices.asciidoc

```diff
@@ -0,0 +1,82 @@
+[role="xpack"]
+[testenv="basic"]
+[[frozen-indices]]
+= Frozen Indices
+
+Elasticsearch indices can require a significant amount of memory available in order to be open and searchable. Yet, not all indices need
+to be writable at the same time and have different access patters over time. For example indices in the time-series or logging use-case
+are unlikely to be queried once they age out but still need to be kept around for retention policy purposes.
+
+In order to keep indices around for a longer time period but at the same time reducing their hardware requirements they can be transitioned
+into a frozen state. Once an index is frozen, all it's transient shard memory aside of mappings and and necessary in-memory structures for
+analysis is moved to persistent storage. This allows for a much higher disk to heap storage ratio on individual nodes. Once an index is
+frozen their are read-only and require their data-structures required for query execution to be loaded on demand. A search request that hits
+one or more shards from frozen indices will execute it's search through a throttling component that ensures that we never search more than
+`N` (`1` by default) searches concurrently. This protects nodes from exceeding the available memory due to incoming search requests.
+
+In contrast to ordinary open indices, frozen indices are expected to execute slow and are not designed for high query load. Parallelism is
+gained only on a per-node level and loading data-structures on demand is expected to be one or more orders of a magnitude slower than query
+execution on a per shard level. Depending on the index and it's data executing a search on a frozen index is expected to be in the seconds
+or even minutes range compared to milliseconds on a non-frozen index.
+
+== Best Practices
+
+Since frozen indices provide a much higher disk to heap ratio for the expense of latency it is recommended to allocate frozen indices on
+dedicated nodes to prevent searches on frozen indices influencing traffic on low latency nodes. There is a significant overhead in loading
+data-structures on demand which can cause page-faults and garbage collections further slowing down query execution.
+
+Since indices that are eligible for freezing are likely to not change in the future disk space can be optimized. Many strategies are
```

Comment:
```suggestion
Since indices that are eligible for freezing are unlikely to change in the future, disk space can be optimized as described in <<tune-for-disk-usage>>.
```